modules/organization/variables.tf

```diff
@@ -14,6 +14,18 @@
  * limitations under the License.
  */
 
+variable "access_policy_title" {
```

Comment:
Is there a way to make this more similar to other modules, eg VPN and NAT with router, where you have an xxxx_create boolean variable that decides whether or not to create the resource? And then a single variable for the resource attributes. I strongly dislike having n variables related to a specific feature, it makes the variable namespace messy.

Reply:
I'll remove the option to create / not create, I will add in a separate CL once we have data source for 'google_access_context_manager_access_policy'